actionkit/api/user.py

```diff
@@ -48,6 +52,73 @@ def get_user(self, user_id):
         return {'res': res,
                 'user': res.json() if res.status_code == 200 else None}
 
+    def update_user(self, user_id, update_dict):
+        res = self.client.patch(
+            #the '/' at the end is IMPORTANT!
+            '%s/rest/v1/user/%s/' % (self.base_url, user_id),
+            data=json.dumps(update_dict))
+        return self._http_return(res)
+
+    def user_search(self, **kwargs):
+        res = ''
+        if kwargs['user_id']:
+            res = self.client.get('%s/rest/v1/user/%s/' % (self.base_url, kwargs['user_id']))
+        else:
+            search_string = '?'
+            search_array = []
+            for key in kwargs:
+                if kwargs[key]:
+                    item_search_string = key + '=' + kwargs[key]
+                    search_array.append(item_search_string)
+            amp = '&'
+            search_string += amp.join(search_array)
+            # for .... reasons... this can NOT have a trailing slash
+            res = self.client.get('%s/rest/v1/user/%s' % (self.base_url, search_string))
+        return {'status_code': res.status_code, 'user': res.json()}
+
+    def get_orders(self, user_id=None, orders_url=None):
+        if not orders_url:
```

Comment:
Alternatively, you could put together the orders_url string with just the user_id; it's always /rest/v1/orders/?user=<user_id> . Like you do with user_url in the add_phone method below.